assets/js/src/core/modules/execution-engine/index.ts

```diff
@@ -17,13 +17,15 @@ import { type JobComponentRegistry } from './services/job-component-registry'
 import { serviceIds } from '@Pimcore/app/config/services'
 import { NotificationJobContainer as DefaultJobContainer } from './jobs/default/notification-job-container'
 import { NotificationJobContainer as DownloadJobContainer } from './jobs/download/notification-job-container'
+import { NotificationJobContainer as ZipUploadJobContainer } from './jobs/zip-upload/notification-job-container'
 
 export const executionEngineModule: AbstractModule = {
   onInit () {
     const jobComponentRegistry = container.get<JobComponentRegistry>(serviceIds['ExecutionEngine/JobComponentRegistry'])
 
     jobComponentRegistry.registerComponent('default', DefaultJobContainer)
     jobComponentRegistry.registerComponent('download', DownloadJobContainer)
+    jobComponentRegistry.registerComponent('unpack', ZipUploadJobContainer)
```

Comment:
Can we rename the type to zip-upload. For me it would be more consistent.

Reply:
done with https://github.com/pimcore/studio-ui-bundle/pull/549/commits/f8356a8313ad21c7e4496d834db5eb15fa906ed4